components/gitpod-protocol/src/workspace-cluster.ts

```diff
@@ -15,6 +15,9 @@ export interface WorkspaceCluster {
     // Must be identical to the installationShortname of the cluster it represents!
     name: string;
 
+    // The name of the application cluster to which this cluster should be registered.
```

Comment:
:bulb: If we decide for non-255 chars we could document the different length-limit here.